features/step_definitions/account_creation.rb

```diff
@@ -10,54 +10,61 @@
   expect(page).to have_content "Log in"
 end
 
-When(/^I click "(.*?)"$/) do |arg1|
-  pending # express the regexp above with the code you wish you had
+When(/^I click the "(.*?)" link$/) do |arg1|
+  click_link arg1 
 end
 
-When(/^I click the button "(.*?)"$/) do |arg1|
+When(/^I click the "(.*?)" button$/) do |arg1|
   click_button arg1
 end
 
 When(/^I enter my email address$/) do
-  pending # express the regexp above with the code you wish you had
+ fill_in :user_email, with: 'test@example.com' 
```

Comment:
This rests on the precondition that you have an `@user` initialized.

There are a few ways to do this but the best way to do it is to initialize that @user variable in your `Given I do not have an account on the site` step when you first create the user account that you are going to use to auth into. Instance variables are accessible across step files.